packages/monitor-v2/src/llm-bot/OptimisticOracleV2.ts

```diff
@@ -150,3 +189,32 @@ export class OptimisticOracleClientV2FilterDisputeable
     });
   }
 }
+
+export class DisputerStrategy {
+  static process(request: OptimisticOracleRequest): Promise<OptimisticOracleRequestDisputable> {
+    return Promise.resolve(
```

Comment:
this is strange why not make the function async and return directly? also what is the point of this class, is it a mock or something?